apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -0,0 +1,1252 @@
+                        key={filter.field}
+                        id={filter.field}
+                        label={n(
+                          filter.field,
+                          TranslationDefaults[filter.field],
+                        )}
+                        labelUse="p"
+                        labelVariant="h5"
+                        iconVariant="small"
+                        expanded={isOpen[index]}
+                        onToggle={() => toggleIsOpen(index, !isOpen[index])}
+                      >
+                        <Stack space={[1, 1, 2]}>
+                          {filter.options.map((option) => {
+                            let keyField = option
+                            let str = filter.field as keyof typeof filters
+
+                            if (str === 'universityId') {
+                              keyField = universities.filter(
+                                (x) => x.id === option,
+                              )[0].title
+                            }
+                            if (keyField !== 'OTHER') {
+                              return (
+                                <Checkbox
+                                  label={`${n(keyField, keyField)}${
+                                    filter.field === 'durationInYears'
+                                      ? locale === 'en'
+                                        ? keyField === '1'
+                                          ? ' year'
+                                          : ' years'
+                                        : ' ár'
+                                      : ''
+                                  }`}
+                                  id={keyField}
+                                  value={option}
+                                  checked={
+                                    filters[str].filter((x) => x === option)
+                                      .length > 0
+                                  }
+                                  onChange={(e) =>
+                                    checkboxEventHandler(e, filter.field)
+                                  }
+                                />
+                              )
+                            }
+                          })}
+                        </Stack>
+                        <Box
+                          display="flex"
+                          width="full"
+                          flexDirection="row"
+                          justifyContent="flexEnd"
+                          marginTop={1}
+                        >
+                          <Button
+                            variant="text"
+                            icon="reload"
+                            size="small"
+                            onClick={() =>
+                              setFilters({ ...filters, [filter.field]: [] })
+                            }
+                          >
+                            {n('clearFilter', 'Hreinsa val')}
```

Comment:
A super nitpick consideration.
Do we want to use `sía` when talking about filters in icelandic?

Reply:
We are using sía but this only refers to a certain subset of filters, not the entire filter so it would be confusing to say "hreinsa síu"